src/Command/CommandHelper.php

```diff
@@ -159,7 +159,7 @@ public static function begin(
 			})($autoloadFile);
 		}
 		if ($projectConfigFile === null) {
-			foreach (['phpstan.neon', 'phpstan.neon.dist', 'phpstan.dist.neon'] as $discoverableConfigName) {
+			foreach (['phpstan.neon', 'phpstan.neon.dist', 'phpstan.dist.neon', 'phpstan.php', 'phpstan.php.dist', 'phpstan.dist.php', 'phpstan.neon.php', 'phpstan.neon.php.dist', 'phpstan.dist.neon.php'] as $discoverableConfigName) {
```

Comment:
`*.dist` files are configured to be NEON files, so it doesn't work with `phpstan.php.dist` and `phpstan.neon.php.dist`.
It will require a bigger refactor to support them, so I suggest to not autodiscover them, would you be OK?